commands/sandbox.go

```diff
@@ -39,7 +39,7 @@ const (
 	// Minimum required version of the sandbox plugin code.  The first part is
 	// the version of the incorporated Nimbella CLI and the second part is the
 	// version of the bridge code in the sandbox plugin repository.
-	minSandboxVersion = "3.0.0-1.1.0"
+	minSandboxVersion = "3.0.2-1.2.0"
```

Comment:
The changes in this PR would be ineffective without changes in dependencies.
1.  The library interface to Nimbella CLI is amended so that certain errors that are not actually thrown by nim but only printed are captured in a special category.   This is needed so that doctl can detect that errors actually occurred in these cases.   Before, the error messages were displaying but the exit from the command was treated as "normal" (exit code 0).
2. The plugin bridge code is amended to feed back the result of (1) in the existing `error` field.